serverless/install/installing-knative-eventing.adoc

```diff
@@ -7,9 +7,9 @@ include::_attributes/serverless-document-attributes.adoc[]
 
 toc::[]
 
-After you install the {ServerlessOperatorName}, you can install Knative Eventing by following the procedures described in this guide.
+Installing Knative Eventing allows you to use event-driven architecture on your cluster. You can create Knative components such as event sources, brokers, and channels, and then use these to send events to your applications.
```

Comment:
Not just sending events to applications. Also sending events to external systems or other applications.

Maybe be more generic like "building workflows or pipelines"